### DISCUSSION TITLE
Unable to find documentation

### DISCUSSION THREAD

**Discussion by anay-p** [primefaces/community#2999] [PrimeNG] [answered]
Hey guys. I am new to Prime, so I apologize if I'm missing something obvious.

I was looking at the PrimeNG and PrimeFlex documentation but couldn't find anything about `layout-wrapper`, `layout-sidebar`, `layout-main-container`, `layout-main`, etc. classes used in the Sakai template. While having the ready-made components is great, structuring the webpage so that the components work cohesively with each other is where I find myself struggling. I think these classes are the key here, but again, not having the documentation makes things very difficult.

Another class I noticed being used repeatedly in the PrimeNG docs is the `card` class, but again, I can't find anything about it.

Thanks in advance for helping out a newbie!

**Comment by JuncalLP** [comment, marked as answer]
Those are not part of PrimeNG or PrimeFlex but from the Sakai template, you can find them in it's assets folder:

https://github.com/primefaces/sakai-ng/blob/master/src/assets/layout/styles/layout/layout.css

**Reply by anay-p** [reply]
I see. Thanks for letting me know.

While we're at it, what's the recommended way or an easy way to structure a webpage using PrimeNG components? I've found myself struggling with just trying to throw the components together in divs.

**Reply by anay-p** [reply]
Let me ask this as a separate question
